src/libYARP_OS/src/Network.cpp

```diff
@@ -871,6 +876,36 @@ bool NetworkBase::getConnectionQos(const ConstString& src, const ConstString& de
     return true;
 }
 
+bool NetworkBase::isValidPortName(const ConstString& portName)
```

Comment:
This is a suggestion about this function, maybe @drdanz and @traversaro can provide their opinions as well 😄 
It would be nice to design this method to also return a `const ConstString& what` so that proper diagnostic can be made in the future in the case we adopt much more detailed/specific policies about names. This will go in the direction of providing users which much more detailed information about errors/failures.

Reply:
I don't know if this is a good idea or not, but if we add this, I don't think it should be a mandatory parameter. Also it cannot be "const" if it should be returned...

Reply:
> I don't know if this is a good idea or not, but if we add this, I don't think it should be a mandatory parameter.

That's ok, we can make a `isValidPortName()` without second argument.

> Also it cannot be "const" if it should be returned...

That is a copy-paste error 🙃, my bad eheh.

Reply:
@Nicogene this was my last comment about this PR. We can avoid extra effort on `isValidPortName()` for the time being.